dev/src/Split/ReleaseNotes.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Dev\Split;
+
+/**
+ * Get release notes for a component
+ */
+class ReleaseNotes
+{
+    /**
+     * @var array
+     */
+    private $changelog;
+
+    /**
+     * @param string $changelog The full release changelog.
+     */
+    public function __construct($changelog)
+    {
+        $this->parseChangelog($changelog);
+    }
+
+    /**
+     * Get release notes for a component.
+     *
+     * @param string $component The component ID
+     * @return string|null
+     */
+    public function get($component)
+    {
+        return isset($this->changelog[$component])
+            ? $this->changelog[$component]
+            : null;
+    }
+
+    private function parseChangelog($changelog)
```

Comment:
So this parses each component (e.g. `Vision`, `SecretManager`) from the changelog and puts the contents of each one in an associative array? Excellent!!

Any way we could get a few comments explaining this?